src/Components/Pages.tsx

```diff
@@ -42,6 +48,70 @@ function getServicesScene(routeMatch: OptionalRouteMatch) {
   });
 }
 
+function EmbeddedSceneWrapper(props: EmbeddedLogsExplorationProps) {
+  // Component is always null, doesn't look like we can embed something from the same app?
+  const { component: LogsDrilldownComponent, isLoading } = usePluginComponent<EmbeddedLogsExplorationProps>(
+    'grafana-lokiexplore-app/embedded-logs-exploration/v1'
+  );
+
+  // We don't want to re-render the entire app every time the props change, only once when the plugin component is done loading
+  // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
This code is a bit odd, becasue when loading is true, you would be returning LogsDrilldownComponent, only to be saved by the condition below.